website/static/js/signUp.js

```diff
@@ -103,6 +103,10 @@ var ViewModel = function(submitUrl, campaign) {
     };
 
     self.submit = function() {
+        if (self.submitted()) {
+            $osf.growl('Already submitted', 'You cannot sign up more than once.', 'error');
```

Comment:
Actually instead of a growlbox this error message can go in the form and replace the "Registration successful" message